[ResponseOps][Cases][9.1 & Serverless] New case analytics indices (#2220)
elastic/docs-content: explore-analyze/alerts-cases/cases/cases-as-data.md

---
applies_to:
  stack: preview 9.2
  serverless: unavailable
---

# Use cases as data [use-cases-as-data]

The cases as data feature lets you visualize data about cases in your [space](/deploy-manage/manage-spaces.md). After turning it on, you can query case data from dedicated case analytics indices and build dashboards and visualizations to track case trends and operational metrics. This information is particularly useful when reporting on key performance indicators (KPIs) such as Mean Time To Respond (MTTR), case severity trends, and analyst workload.

::::{admonition} Requirements
To use cases as data, you must have the appropriate subscription. Refer to the subscription page for [Elastic Cloud](https://www.elastic.co/subscriptions/cloud) and [Elastic Stack/self-managed](https://www.elastic.co/subscriptions) for the breakdown of available features and their associated subscription tiers.
::::

## Turn on cases as data [turn-on-cases-as-data]

> To turn on cases as data, add `xpack.cases.incrementalId.enabled: true` to your [`kibana.yml`](/deploy-manage/stack-settings.md) file.

Author: ```suggestion
To turn on cases as data, add `xpack.cases.analytics.index.enabled: true` to your [`kibana.yml`](/deploy-manage/stack-settings.md) file.
```

::::{warning} 

> If your deployment has more than 10 spaces, we do not recommend turning on cases as data. The background tasks that refresh the case analytics indices in each space are run every five minutes and may overload Task Manager.  

Author: ```suggestion
We only recommend turning this feature on if your deployment has 10 or fewer spaces with cases. The background tasks that refresh the case analytics indices in each space are run every five minutes and may overload Task Manager.  
```
